Frameworks/SystemConfiguration/SCNetworkReachability.mm

```diff
@@ -134,15 +134,19 @@ + (void)_testAllReachabilityObjects {
 
 + (void)_checkGlobalReachability {
     WNCConnectionProfile* internetConnectionProfile = [WNCNetworkInformation getInternetConnectionProfile];
-    if (internetConnectionProfile &&
-        [internetConnectionProfile getNetworkConnectivityLevel] == WNCNetworkConnectivityLevelInternetAccess) {
-        SCNetworkReachabilityFlags newFlags = kSCNetworkReachabilityFlagsReachable;
+    @try {
+        if (internetConnectionProfile &&
+            [internetConnectionProfile getNetworkConnectivityLevel] == WNCNetworkConnectivityLevelInternetAccess) {
+            SCNetworkReachabilityFlags newFlags = kSCNetworkReachabilityFlagsReachable;
 
-        if (internetConnectionProfile.isWwanConnectionProfile) {
-            newFlags |= kSCNetworkReachabilityFlagsIsWWAN;
+            if (internetConnectionProfile.isWwanConnectionProfile) {
+                newFlags |= kSCNetworkReachabilityFlagsIsWWAN;
+            }
+            _globalReachableFlags = newFlags;
+        } else {
+            _globalReachableFlags = 0;
         }
-        _globalReachableFlags = newFlags;
-    } else {
+    } @catch (NSException* exception) {
```

Comment:
Bummer given other places have this too.  I've opened #1796 to track other instances of this.